TestProject/UnitTest1.cs

```diff
@@ -2,55 +2,22 @@
 namespace TestProject {
    [TestClass]
    public class UnitTest1 {
+      MyDouble mMyDouble = new ();

```

Comment:
Declare private variables at the end of the program. Rename mMyDouble => mDouble.